pmm/pmm2-dbaas-e2e-tests.groovy

```diff
@@ -65,6 +65,10 @@ pipeline {
             defaultValue: 'dev-latest',
             description: 'PMM Client version',
             name: 'CLIENT_VERSION')
+        string(
+            defaultValue: '-e PMM_DEBUG=1 -e ENABLE_DBAAS=1 -e PERCONA_TEST_VERSION_SERVICE_URL=https://check-dev.percona.com/versions/v1 -e PERCONA_TEST_DBAAS_PMM_CLIENT=perconalab/pmm-client:dev-latest',
```

Comment:
Why hardcode this? We could use environment variable to set this value, so user could pass custom value?

Reply:
Wdym? This was hardcoded before (-e PMM_DEBUG=1 -e ENABLE_DBAAS=1 -e PERCONA_TEST_VERSION_SERVICE_URL=https://check-dev.percona.com/versions/v1) were always used when 
pmm2-dbaas-e2e-tests job was triggered. Now we actually have the option to change the env variables as you want. Just putting PERCONA_TEST_DBAAS_PMM_CLIENT=perconalab/pmm-client:dev-latest as default because it makes sense with default dev-latest server.
![Screenshot from 2022-07-29 15-12-30](https://user-images.githubusercontent.com/64002801/181768359-7e4350ee-1988-4c73-9b8a-55ac3b5ee215.png)

Reply:
Sorry I was thinking of having a seperate client docker tag field and using that, but it is going to over complicate, please go ahead